otredis.go

```diff
@@ -2,34 +2,68 @@ package otredis
 
 import (
 	"context"
+	"fmt"
+	"strings"
 
 	"github.com/go-redis/redis"
-	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 )
 
-// WrapRedisClient adds opentracing measurements for commands and returns cloned client
-func WrapRedisClient(ctx context.Context, c *redis.Client) *redis.Client {
+func WrapWithOpenTracing(ctx context.Context, client *redis.Client) *redis.Client {
```

Comment:
We should probably keep the same function name (for better compatibility)

Reply:
Yep, I see that after my 1st commit and already fix it :smile: